data/README.md

```diff
@@ -3,16 +3,16 @@
 If you go to the "RESULTS" section of your job, you can download results.
 
 Aggregated reports
-------------------------------
+------------------
```

Comment:
Just for my edification, what's the story here? The resulting markdown output is the same.